docs/algorithms.rst

```diff
@@ -438,6 +446,8 @@ where the pseudoenergy :math:`u_*(X)` for the replica-exchange simulation is def
 That is, :math:`u_*(X)` is the sum of the reduced potential energies of each replica configuration at the current
 thermodynamic state it is visiting.
 
+.. _autocorrelate_algorithm:
+
 Determining equilibration and autocorrelation in a replica-exchange simulation
```

Comment:
Change to "Identifying equilibration and production regions in a replica-exchange simulations"